release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -170,6 +170,11 @@ Previously, you could not specify the subdomain of a route, and the `spec.host`
 
 You can use this enhancement to simplify sharding by enabling a route to have multiple, distinct host names determined by each router deployment that exposes the route.
 
+[id="ocp-4-11-external-dns-operator"]
+==== External DNS Operator is GA in {product-title} 4.11
+
+The External DNS Operator is GA for AWS Route53, Azure DNS, GCP DNS, and Infoblox. The External DNS Operator is TechPreview for BlueCat and AWS Route53 on GovCloud. You can now create DNS records on a public DNS zone on Infoblox by using the External DNS Operator. Previously, you needed to create a namespace for the operands and some RBAC prior the installation of the External DNS Operator. By default, the External DNS Operator deploys the operands in the namespace `external-dns-operator`. You must migrate from TechPreview to GA. The upstream version for an {product-title} 4.11 is `v0.12.0` and for the TechPreview was `v0.10.2`. For more information, see xref:../networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc#nw-installing-external-dns-operator-on-cloud-providers[About the External DNS Operator].
```

Comment:
Sorry forgot about that but we need to add some details on the changes of the upstream version. So the new version contains the following main changes:

- Status of the routes source is now used to retrieve the DNS FQDN names (useful in regard of the route subdomain feature)
- BlueCat DNS provider now allows the users to enable the automatic DNS config deployment
- Proxy support for BlueCat DNS provider

Reply:
Addressed